### ISSUE TITLE
Use node template

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/js-context-this#8]
Originally "Change examples in lab to run in node instead of the browser"

**Comment by RealWeeks** [comment]
Parts of this lab require the developer learning about the dom

**Comment by jrhorn424** [comment]
Yes, the labs need extensive work. Since this is node week, the labs should be re-written to use node instead of the browser.

**Comment by RealWeeks** [comment]
I wish I had seen this issue earlier, this is on me.

**Comment by raq929** [comment]
For cohort 015, this is the first node lesson, before they've had node-nodeschool-study.
@payne-chris-r and I think it makes sense to not use node yet.

**Comment by gaand** [comment]
No.  If the calendar doesn't work then we should fix the calendar.

@ga-wdi-boston/core

**Comment by jrhorn424** [comment]
The labs here are not canon. They are legacy. They need to be broken. They focus too much on the DOM to the detriment of scope and context.

See #36 for concerns regarding scaffolding.

**Comment by payne-chris-r** [comment]
I think it DOES make sense to use node for this lesson. I was either confused or wrong if previously said it doesn't make sense to use node yet. They've used node in the terminal since `js-basics`. I don't see any reason why it's confusing in this lesson. It's what I've done in the past in this lesson with the exception of the labs.

**Comment by raq929** [comment]
Oh, I was thinking this comment was about using it for the labs.
